cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ClusterPartitionState.java

```diff
@@ -112,7 +132,7 @@ protected void populateKafkaPartitionState(Set<PartitionInfo> underReplicatedPar
         }
         for (PartitionInfo partitionInfo : _kafkaCluster.partitionsForTopic(topic)) {
           int numInsyncReplicas = partitionInfo.inSyncReplicas().length;
```

Comment:
Do we still need this var?

Reply:
Yes, we still need it to check if th3e current Partition is underMinIsrPartitions

Reply:
Resolving this as of Now.